test/pubspec_overrides_test.dart

```diff
@@ -22,12 +23,14 @@ void main() {
         }),
       ]).create();
 
+      final overridesPath = p.join('.', 'pubspec_overrides.yaml');
       await pubCommand(
         command,
-        warning:
-            'Warning: pubspec.yaml has overrides from pubspec_overrides.yaml\n'
-            'Warning: You are using these overridden dependencies:\n'
-            '! lib 2.0.0',
+        output: contains(
+          '''
+! lib 2.0.0 (overridden in $overridesPath)
```

Comment:
Let's add another test cases where:
 * `foo` and `bar` is overriden in `pubspec.yaml`
 * `foo` is overriden in `pubspec_override.yaml`

Then we should check for:
```
! foo 2.0.0 (overridden in $overridesPath)
! bar 2.0.0 (overridden)
```

Reply:
Added a case (but the override of foo is shadowed)